src/main/kotlin/lotto/main.kt

```diff
@@ -0,0 +1,21 @@
+package lotto
+
+import lotto.domain.Lotto
+import lotto.domain.Money
+import lotto.domain.Winner
+import lotto.view.InputView
+import lotto.view.OutputView
+
+fun main() {
+    val lotto = Lotto()
+    val money = Money(InputView.getPurchaseFee())
+    lotto.setCountByPurchaseFee(money.getPurchaseFee())
+    val tickets = lotto.purchaseTicket()
+    OutputView.printTicketPurchaseCount(lotto.getCount())
+    OutputView.printTicketsInfo(tickets)
+    val winner = Winner(InputView.getWinNumbers())
+    val matchInfo = winner.checkNumberMatch(tickets)
+    OutputView.printStatistics(matchInfo)
+    money.setRevenueRate(matchInfo)
+    OutputView.printRevenueRate(money)
```

Comment:
가독성을 의미가 달라지는 부분에서 위해 띄어쓰기를 해주세요 :) 
```suggestion
    val lotto = Lotto()
    val money = Money(InputView.getPurchaseFee())
    
    lotto.setCountByPurchaseFee(money.getPurchaseFee())
    
    val tickets = lotto.purchaseTicket()
    
    OutputView.printTicketPurchaseCount(lotto.getCount())
    OutputView.printTicketsInfo(tickets)
    
    val winner = Winner(InputView.getWinNumbers())
    val matchInfo = winner.checkNumberMatch(tickets)
    
    OutputView.printStatistics(matchInfo)
    
    money.setRevenueRate(matchInfo)
    
    OutputView.printRevenueRate(money)
```